receiver/receivercreator/README.md

```diff
@@ -1,12 +1,68 @@
 # receiver_creator
 
-This receiver can instantiate other receivers at runtime based on whether observed endpoints match a configured rule.
+This receiver can instantiate other receivers at runtime based on whether observed endpoints match a configured rule. These rules are evaluated for each endpoint discovered. If the rule evaluates to true then the receiver for that rule will be started against the matched endpoint.
 
-## TODO
-* Observer endpoint details
-* Rule matching
+## Config
+
+**watch_observers**
+
+A list of observers previously defined to be run in the `extensions` section. receiver_creator will watch for endpoints generated by these observers.
+
+**receivers**
+
+A map of receiver names (e.g. `redis/1`) to a template for when and how to instantiate that receiver.
+
+**receivers.&lt;receiver_type/id&gt;.rule**
+
+Rule string using [expvar syntax](https://github.com/antonmedv/expr/blob/master/docs/Language-Definition.md). Variables available are detailed below in [Endpoints](#endpoints).
+
+Each rule must start with `type.(pod|port) &&` such that the rule matches only one endpoint type.
```

Comment:
Consider moving this into the Endpoints section below (which might be better titled "Rule Expressions").